src/parsec.rs

```diff
@@ -244,6 +244,22 @@ impl ParsecMap {
             .flatten()
     }
 
+    #[cfg(feature = "mock_parsec")]
+    pub fn unpolled_observations_string(&self) -> String {
+        let parsec = if let Some(parsec) = self.map.values().last() {
+            parsec
+        } else {
+            return String::new();
+        };
+
+        parsec.unpolled_observations_string()
+    }
+
+    #[cfg(all(not(feature = "mock_parsec"), feature = "mock_base"))]
```

Comment:
will `#[cfg(not(feature = "mock_parsec"))]` be enough?

Reply:
Because that function is only use for test and so only available for mock base (might have been clippy complaining).